chpl/chpl-resources/src/main/resources-dev/jobs.xml

```diff
@@ -303,7 +303,7 @@
                     </entry>
                     <entry>
                         <key>acb</key>
-                        <value>Drummond Group☺ICSA Labs☺InfoGard☺SLI Compliance</value>
+                        <value>Drummond Group☺ICSA Labs☺UL LLC☺SLI Compliance</value>
```

Comment:
It's interesting that you had to change these jobs files. The change you made in the website project pretty much gets us to the point where the website will keep working fine and display data as expected (barring stale cache data) if an ACB changes their name. The scheduled jobs however would not keep working. I think it's a little dangerous to even let them change their name via the UI edit dialog because of that. Suggest revisiting how we handle ACB permissions in the scheduled jobs (in a different and larger ticket probably) since theoretically any ACB could go in and change their name at any time.. even if something small like adding punctuation.  We could live with old ACB names in comments and unit tests because that wouldn't actually break anything but it would here in the jobs. Another note - as I was thinking about this I looked at the quartz jobs tables and noted that the ACB name is appended to the trigger name, so we have triggers in prod like "stephanie_eckgren@ul_comInfoGard". Does that need to be changed? Is it used anywhere in any way that it would matter?